src/common-tests.js

```diff
@@ -20,7 +20,8 @@ function common (app, errors, serviceName = 'people', idProp = 'id') {
     );
 
     it('sets `events` property from options', () =>
-      expect(app.service(serviceName).events).to.deep.equal([ 'testing' ])
+      expect(app.service(serviceName).events.indexOf('testing'))
+        .to.not.equal(-1)
```

Comment:
It's checking for Array index in case we already have existing events (the case e.g. in RethinkDB)